core/loadBalancer/src/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -69,8 +69,9 @@ class LoadBalancerService(config: WhiskConfig, verbosity: Verbosity.Level)(
         { () =>
             val producedCount = getActivationCount()
             val invokerInfo = invokerHealth.getInvokerActivationCounts()
-            val consumedCounts = invokerInfo map { indexCount => indexCount._2 }
-            val consumedCount = consumedCounts.foldLeft(0)(_ + _)
+            val consumedCount = invokerInfo map {
+                case (index, count) => count
```

Comment:
You can scalaify even more with `case (_, count)` ;) (or really, just `invokerInfo.map(_._2).sum`)